src/lnd.rs

```diff
@@ -173,6 +173,125 @@ impl LndClient {
         Ok(())
     }
 
+    /// Creates a fully populated PSBT with 1 output to the specified address of the
+    /// specified amount. Coin selection is performed automatically. After either verifying the
+    /// inputs, all input UTXOs are locked with an internal LND app ID.
+    ///
+    /// NOTE: If this method returns without an error, it is the caller's responsibility to either
+    /// spend the locked UTXOs (by finalizing and then publishing the transaction) or to
+    /// unlock/release the locked UTXOs in case of an error on the caller's side.
+    pub async fn fund_original_psbt(
+        &self,
+        address: &bitcoin::Address,
+        amount: bitcoin::Amount,
+    ) -> Result<(PartiallySignedTransaction, Vec<UtxoLease>), LndError> {
+        use tonic_lnd::walletrpc::fund_psbt_request::{Fees, Template};
```

Comment:
Curious why there `use` statements are inside the function rather than top of file, idk enough rust to know if this changes any behaviour

Reply:
this scopes the import to this function. We don't use `Fees` or `Template` or anything from `fund_psbt_request` outside. Smaller scopes make it easer to find why an import is used.